CHANGELOG.md

```diff
@@ -29,6 +29,14 @@ You can check your current version with the following command:
     ```
 
 For more information, see [UP42 Python package description](https://pypi.org/project/up42-py/).
+
+## 0.37.2a1
```

Comment:
we need to update version.
This needs to be 0.37.3a1

Reply:
No changelog update is needed, since there are no changes to the code